### ISSUE TITLE
Lakebtc createorder incorrect signature

### ISSUE THREAD

**Issue by awsreck** [ccxt/ccxt#6830]
Hi 
using lakebtc i can fetchBalance but can't create order , i receive 
https://api.lakebtc.com/api_v2 POST 401 {"error":"Signature xxxxx is incorrect."}

can't see what's wrong 
Thanks

**Comment by kroitor** [comment]
Hi! Unfortunately, LakeBTC does not let me in. So, I currently don't have a working keypair. If you can share an empty keypair for testing and send it to igor.kroitor@gmail.com or info@ccxt.trade – we could then speed up the fix and resolve this issue quickly. We don't need your funds or your data, so, a zero balance keypair would really help, this for debugging purposes only. If not - no worries, we will restore the access to it soon.

**Comment by kroitor** [comment]
@awsreck thx for your assistance, this bug is fixed in CCXT version 1.26.74 (the new build will arrive in 15 minutes). It will be great if you report back whether it works on your side or not. I have tested it extensively on my side – and it worked for me. Apart from that I've also added some better error handling to it. Let me know if it works as expected for you, or if you experience any further difficulties with it. Thx again!
